web/src/pages/index.md

```diff
@@ -29,14 +29,13 @@ The design and development of the smart air hockey table will adhere to the
 following focused design requirements to ensure a successful and captivating
 gameplay experience:
 
-1. PSDR #1 (Hardware): An ability to control a grid of lights beneath the
-   playing surface to create complex visuals.
-2. PSDR #2 (Hardware & Software): An ability to accurately track the puck’s
-   position on the playing field in real time.
-3. PSDR #3 (Hardware): An ability to reliably detect goals scored by monitoring
-   the passage of the puck through the goal.
-4. PSDR #4 (Hardware): An ability to internally track the score of ongoing
-   games, which is then displayed with a display.
-5. PSDR #5 (Software): An ability to display dynamic light effects on the
-   playing surface based on game state by implementing software that synchronizes
-   isual effects with ongoing gameplay.
+1. PSDR #1 (Hardware): 
+An ability to convert analog hall effect sensor data into digital signals with custom threshold values using comparators.
+2. PSDR #2 (Hardware & Software):
+An ability to implement custom digital logic to convert digital hall effect sensor data into precise puck row/column position data.
+3. PSDR #3 (Hardware):
+An ability to reliably detect goals scored by monitoring the passage of the puck through the goal using a photoresistor and LED pair.
+4. PSDR #4 (Hardware):
+An ability to display game and debug information on an OLED display connected via the SPI communication protocol.
```

Comment:
remove debug info reference, not sure its "testable" by the course staff

Reply:
Possibly make "game info" more specific as well